templates/default/change_owner.css

```diff
@@ -1,11 +1,40 @@
-.xoct_invite, .xoct_remove {
-    position: absolute;
-    top: 6px;
-    right: 5px;
-    padding: 5px 8px !important;
-}
+
```

Comment:
Unwanted new line! Please remove it and keep it at max 1 new line so that it would look better.